PWGLF/Tasks/Strangeness/derivedcascadeanalysis.cxx

```diff
@@ -605,6 +617,11 @@ struct derivedCascadeAnalysis {
       auto poseta = RecoDecay::eta(std::array{casc.pxpos(), casc.pypos(), casc.pzpos()});
       auto negeta = RecoDecay::eta(std::array{casc.pxneg(), casc.pyneg(), casc.pzneg()});
       auto bacheta = RecoDecay::eta(std::array{casc.pxbach(), casc.pybach(), casc.pzbach()});
+
+      auto fullMomentumPosDaugh = TMath::Sqrt(TMath::Power(casc.pxpos(), 2) + TMath::Power(casc.pypos(), 2) + TMath::Power(casc.pzpos(), 2));
```

Comment:
If you define `auto pVecPos = std::array{casc.pxpos(), casc.pypos(), casc.pzpos()}`, you can directly use
```c++
auto poseta = RecoDecay::eta(pVecPos);
auto fullMomentumPosDaugh = RecoDecay::p(pVecPos);
```
Even better would be to add a dynamic column that returns `std::array{casc.pxpos(), casc.pypos(), casc.pzpos()}`.
See https://github.com/AliceO2Group/O2Physics/blob/4ecf524826c7279cb172e35ac2b390f9696dd84c/PWGHF/DataModel/CandidateReconstructionTables.h#L455-L456